### PR TITLE
Fix Uneventful NPE in MockConsumer

### PR DESCRIPTION
**Pull request by garyrussell**
Adds noise in debug logs.

### CODE REVIEW

**Review by ahmedalnuaimi — APPROVED** [review]
LGTM

**Comment by garyrussell** [comment]
@simonbasle Do you know why this PR still needs more approvals? Is it because @artembilan is not in some "approvers" list?

Thanks.

**Comment by simonbasle** [comment]
@garyrussell ah yes, I've just added @artembilan to the appropriate list

**Pull request merged**
